jaeger-crossdock/docker-compose.yml

```diff
@@ -35,40 +35,74 @@ services:
 
     go:
         image: jaegertracing/xdock-go
+        links:
+            - "jaeger:jaeger-agent"
         ports:
             - "8080-8082"
 
     python:
         image: jaegertracing/xdock-py
+        links:
+            - "jaeger:jaeger-agent"
         ports:
             - "8080-8082"
 
     nodejs:
         image: jaegertracing/xdock-node
+        links:
+            - "jaeger:jaeger-agent"
         ports:
             - "8080-8082"
 
     java-udp:
-        depends_on:
-            - test_driver
         build: .
+        links:
+            - "jaeger:jaeger-agent"
+        depends_on:
+            # Udp sender needs to know agent's address
+            - jaeger
         ports:
             - "8080-8082"
         environment:
             - SENDER=udp
 
     java-http:
         build: .
+        links:
+            - "jaeger:jaeger-agent"
```

Comment:
Did you get the test pass locally? Failure looks specific to the change.

Java http actor would be trying to submit spans to collector, so the renaming probably needs to be to jaeger-collector.

Reply:
Yep, you are right. With suggested change I got it working locally